bluemira/utilities/plot_tools.py

```diff
@@ -121,6 +122,17 @@ def make_gif(folder, figname, formatt="png", clean=True):
     if clean:
         for fp in ims:
             os.remove(fp)
+
+    if len(images) > 1:
+        # Occasionally bbox_tight is a few pixels off for contours it seems
+        # Resize the offending images
+        sizes = np.array([image.shape for image in images])
+        if not (sizes == sizes[0]).all():
+            shape = tuple(sizes[1][:2])
```

Comment:
Not a problem, just curious, is there a reason we're using the size from the second image and not the first?

Reply:
Urgh.. yeah; the first is invariably the offending image. For reasons I don't understand...